crates/wasmtime/src/func.rs

```diff
@@ -676,8 +526,8 @@ impl Func {
     ///     "#,
     /// )?;
     /// let instance = Instance::new(&store, &module, &[add.into()])?;
-    /// let foo = instance.get_func("foo").unwrap().get2::<i32, i32, i32>()?;
-    /// assert_eq!(foo(1, 2)?, 3);
+    /// let foo = instance.get_func("foo").unwrap().typed::<(i32, i32), i32>()?.clone();
```

Comment:
Given this seems to be common (`get_func` followed by `typed` which then needs to be cloned), perhaps we should have `Instance::get_typed_func` to shorthand this?

Reply:
Seems reasonable to me!